iOS Example/iOS Example.xcodeproj/project.pbxproj

```diff
@@ -7,30 +7,34 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		7810985B1EAE77E900FC4CF2 /* RxSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B388AAE91C65640700726BCA /* RxSwift.framework */; };
+		7810985C1EAE77E900FC4CF2 /* RxSwift.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = B388AAE91C65640700726BCA /* RxSwift.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
As you can see this is the iOS Example folder, the project won't run otherwise (it actually didn't compile before)